src/main/java/com/github/crimsondawn45/fabricshieldlib/lib/object/FabricShield.java

```diff
@@ -17,14 +10,4 @@ public interface FabricShield {
     default boolean displayTooltip() {
         return true;
     }
-
-    /**
-     * Adds a tooltip immediately after the name & before the tooltip saying shield stats.
-     *
-     * @param stack   shield's item stack
-     * @param tooltip current tooltip
-     * @param context context
-     */
-    default void appendTooltip(ItemStack stack, Item.TooltipContext context, List<Text> tooltip, TooltipType type) {
```

Comment:
I'm curious about why we introduced the `appendTooltip` callback in the first place.